ddtrace/llmobs/_integrations/mcp.py

```diff
@@ -3,6 +3,9 @@
 from typing import List
 from typing import Optional
 
+from mcp import InitializeRequest
+from mcp.server.streamable_http import MCP_SESSION_ID_HEADER
```

Comment:
although these should be available here, we try to keep a pattern on `try`/`except`ing imports outside of patching files

```suggestion
try:
    from mcp import InitializeRequest
    from mcp.server.streamable_http import MCP_SESSION_ID_HEADER
except:
    InitializeRequest = None
    MCP_SESSION_ID_HEADER = None
```

and then where they're used, doing a `if InitializeRequest is not None`, etc., check